src/App.js

```diff
@@ -1,15 +1,389 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+import './components/Article';
+import Article from "./components/Article";
+import {useForm} from "react-hook-form";
+import logo from './assets/screenshot-logo.png';
+import InputField from "./components/InputField";
+// import {set} from "react-hook-form";
 
 function App() {
-  return (
-    <>
-      <h1>Fruitmand bezorgservice</h1>
-    </>
-  );
+    //fruit counter
+    const [strawberryCounter, setStrawberryCounter] = useState(0);
+    const [bananaCounter, setBananaCounter] = useState(0);
+    const [appleCounter, setAppleCounter] = useState(0);
+    const [kiwiCounter, setKiwiCounter] = useState(0);
+
+    const[clicked, toggleClicked] = React.useState(false);
+
+    const {register} = useForm();
+
+    // const { handleSubmit, formState: { errors }, register, watch } = useForm();
+    // const [formReferrer, setFormReferrer] = useState('anders');
+
+    //contact formula
+    const [firstName, setFirstName] = useState("");
+    const [lastName, setLastName] = useState("");
+    const [age, setAge] = useState(0);
+    const [postcode, setPostcode] = useState("");
```

Comment:
Probeer variabelnamen waar mogelijk in het Engels te formuleren

Reply:
Thanks voor je feedback Zwen, ga ik doen!